README.md

```diff
@@ -30,11 +30,11 @@ To build the local samples and tests you'll have to clone this repository and ru
     ```git clone --recurse-submodules https://github.com/KhronosGroup/Vulkan-Hpp.git```
 2. Change the current directory to the newly created Vulkan-Hpp directory.
 3. Create a build environment with CMake
-    ```cmake -DSAMPLES_BUILD_WITH_LOCAL_VULKAN_HPP=ON -DSAMPLES_BUILD=ON -DTESTS_BUILD_WITH_LOCAL_VULKAN_HPP=ON -DTESTS_BUILD=ON -B build```
+    ```cmake  -DVULKAN_HPP_SAMPLES_BUILD=ON -DVULKAN_HPP_SAMPLES_BUILD_WITH_LOCAL_VULKAN_HPP=ON -DVULKAN_HPP_TESTS_BUILD=ON -DVULKAN_HPP_TESTS_BUILD_ONLY_DYNAMIC=ON -B build```

```

Comment:
We should not list `-DVULKAN_HPP_TESTS_BUILD_ONLY_DYNAMIC=ON` here but -`DVULKAN_HPP_TESTS_BUILD_WITH_LOCAL_VULKAN_HPP=ON`.